components/03-modules/newsletter/_newsletter.scss

```diff
@@ -0,0 +1,94 @@
+$newsletter__font-size                : $font-size-base !default;
+$newsletter__font-size--agreements    : $font-size-small !default;
+$newsletter__field-border-radius      : 100px !default;
```

Comment:
we don't need 100px border-radius. It will be the same result when we will insert here real height of input with paddings (32px)

Reply:
done